Generate_Client/generate.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+
+origin="$(pwd)"
+cd ~
+if [ ! -d "./openapi-generator" ]
+  then
+        git clone https://github.com/openapitools/openapi-generator
+        cd ~/openapi-generator/
+        mvn clean package
+        exit 0
+else
+        cd ~/openapi-generator
+fi
+
+
+dest_dir=${2}
+
+#The following line invokes openapi-generator that generates client library
+#Make sure theres a config.json file in your current working directory
+#Also Do not edit this line..I dont see any reason to..
```

Comment:
This comment is not helpful for future developers who may actually need to edit this file. Please remove it.

Reply:
Ack